packages/block-library/src/rss/index.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Server-side rendering of the `core/rss` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the rss error message.
+ *
+ * @param string $message RSS error message.
+ * @return string|void Void if renders on frontend.
+ */
+function render_rss_error_message( $message ) {
+	return '<div class="components-placeholder"><div class="notice notice-alt notice-error">' . $message . '</div></div>';
```

Comment:
I feel like this extra function is overkill, especially because it is in global namespace. Just add this markup inline.